### PR TITLE
Updates Xcake Action to support  new `xcake make` command.

### PR DESCRIPTION
**Pull request by jcampbell05**
Updates Xcake Action as the command for generating a project in more recent versions changed from `xcake` to `xcake make`

### CODE REVIEW

File: fastlane/docs/Actions.md [hunk 1]
```diff
@@ -289,10 +289,10 @@ xcode_select "/Applications/Xcode6.1.app"
 
 ### [Xcake](https://github.com/jcampbell05/xcake/)
 
-If you use [Xcake](https://github.com/jcampbell05/xcake/) you can use the `xcake` integration to run `xcake` before building your app.
+If you use [Xcake](https://github.com/jcampbell05/xcake/) you can use the `xcake` integration to run `xcake make` before building your app.
 
 ```ruby
-xcake
+xcake make
```
Comment (KrauseFx) [inline fastlane/docs/Actions.md]: What's the space for? Does that work?

**Comment by jcampbell05** [comment]
@krausefx yes as the main functionality is a command called make rather than just running xcake.

**Comment by mfurtak** [comment]
Hm, I think what @KrauseFx is asking is whether `xcake make` works in the context of a `Fastfile`. I think those Ruby blocks in the **Actions.md** are used for describing the syntax of the action, rather than the syntax of the tool the action uses.

I'm coming at this from the side without a lot of knowledge, so apologies if I'm mis-interpreting!

**Comment by jcampbell05** [comment]
Ohhhh I get it, yeah that was a mistake haha

**Comment by KrauseFx** [comment]
Thanks for clarifying @mfurtak

File: fastlane/docs/Actions.md (hunk 1 above)
Comment (KrauseFx) [inline fastlane/docs/Actions.md]: So this needs to be updated

**Comment by KrauseFx** [comment]
Once the docs are updated and commits squashed, this is ready from my side

**Comment by asfalcone** [comment]
@jcampbell05 Just wanted to ping on updating this PR per @KrauseFx's comments so we can get it merged! Thanks!

**Comment by jcampbell05** [comment]
@asfalcone  Thanks for the ping, I'll get this up to date :)

**Comment by jcampbell05** [comment]
@KrauseFx  @mfurtak  @asfalcone  Updated :)

**Comment by asfalcone** [comment]
@jcampbell05 Looks like CI is failing. Can you rebase on master? There were some recent changes to the bundle directory and it looks like your branch doesn't have them. Thanks 👍

**Comment by jcampbell05** [comment]
@asfalcone Sure :) do you have an example command I can run ?

**Comment by asfalcone** [comment]
Should be something like this. Basically you need to get your master up to date and then update your branch accordingly. You can squash while you're in there too :)
```
git fetch upstream master
git checkout f/SupportForNewXcakeCommand
git rebase upstream/master
git rebase -i HEAD~3
git push -f origin f/SupportForNewXcakeCommand
```
http://blogs.atlassian.com/2013/07/git-upstreams-forks/

**Comment by KrauseFx** [comment]
Hey @jcampbell05, what's the status on this?

**Comment by jcampbell05** [comment]
@KrauseFx I'll squash and update.

**Comment by jcampbell05** [comment]
Having issues squashing since I had to merge :)

**Comment by KrauseFx** [comment]
Awesome, thanks @jcampbell05 🚀

**Pull request merged**